api-spec/STAC-standalone.yaml

```diff
@@ -349,6 +362,51 @@ components:
 
         extent or all relevant temporal properties.
       example: '2018-02-12T00:00:00Z/2018-03-18T12:31:12Z'
+    catalogDefinition:
+      type: object
+      properties:
+        name:
+          type: string
+          example: NAIP
+        description:
+          type: string
+          example: Catalog of NAIP Imagery
+        license: 
+          type: string
+          example: PDDL-1.0
+        links: 

```

Comment:
We should add `additionalProperties: true` to allow additional properties like type or title.

Reply:
I'm a little confused on where you wan't me to add additionalProperties here. license is a string and links is an array. I am adding it at the catalogDefinition level which I think is what you wanted on your comment below as well?

Reply:
Was meant for the links object, seems I marked the wrong line, sorry.